.github/workflows/aws-upstream-tests.yml

```diff
@@ -1,75 +0,0 @@
-name: run-upstream-tests
```

Comment:
I guess we'll also need to update the make target [here](https://github.com/pulumi/pulumi-aws/blob/3ff4fa4ddc54bcef6d043e7c62e0e881c6ecdaa5/Makefile#L192). It's deleting all workflows

Reply:
I'll update. Good callout.